usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -39,10 +46,15 @@
     "award_search": {
         "model": AwardSearchView,
         "source_query": award_search_load_sql_string,
+        "source_database": None,
+        "source_table": None,
         "destination_database": "rpt",
+        "swap_table": "vw_award_search",
```

Comment:
I think this should just be `award_search`. [PIPE-291](https://federal-spending-transparency.atlassian.net/browse/PIPE-291) will involve refactoring some code to use the `award_search` table instead of `vw_award_search` view.

Reply:
Yeah I couldn't find an `award_search` postgres table to test with and the model was `AwardSearchView`, so I thought it'd make sense to update `swap_table` alongside the model in PIPE-291. If you still want me to update this value now I could do it.

Reply:
Updating it later as a part of 291 makes sense!